.github/workflows/release.yaml

```diff
@@ -3,12 +3,52 @@ on:
   push:
     tags:
       - "v*"
+  workflow_dispatch: {}
+  schedule:
+    - cron: "45 0 * * *"
 
 env:
   CARGO_TERM_COLOR: always
   RUSTFLAGS: "-D warnings"
+  VERSION: "${{ github.event_name == 'push' && github.ref_name || 'nightly' }}"
+
+permissions:
+  contents: write
+  id-token: write
+  packages: write
 
 jobs:
+  prepare-nightly:
```

Comment:
Could you add a comment explaining what this job is for please 🙏

Reply:
Added.